README.md

```diff
@@ -1657,3 +1658,83 @@ const Person = ({
   // ...
 };
 ```
+## 12. Google Tag Manager
+- 12.1 [What is Google Tag Manager](#12.1-what-is-google-tag-manager)
+- 12.2 [Installations](#12.2-installations)
```

Comment:
Installation might changes depends on the project's stacks. We can remove it.

Reply:
@tassioFront  do you think it's better to update this later, in case it changes?

Reply:
@IngridCavalcantiGti, to be honest, I would remove it. On the LV, for example, we do not use this approach.

Reply:
done @tassioFront  [ffb02f3](https://github.com/juntossomosmais/frontend-guideline/pull/41/commits/ffb02f341eadbc9da4095cee2155c1777cca8069)